lib/actions/lookupObjects.js

```diff
@@ -302,10 +302,10 @@ module.exports.process = async function processAction(message, configuration) {
   this.logger.info(`Got ${response.records.length} records`);
 
   if (configuration.outputMethod === 'emitIndividually') {
-    response.records.forEach((record) => {
-      this.emit('data', messages.newMessageWithBody(record));
+    response.records.forEach(async (record) => {
```

Comment:
It is probably better to use `for (const record of response.records) { ... }` syntax instead of `.forEach(async (record) =>`

Reply:
In general, I agree. But with the current eslint settings it will look like this.
```javascript
    // eslint-disable-next-line no-restricted-syntax
    for (const record of response.records) {
      // eslint-disable-next-line no-await-in-loop
      await this.emit('data', messages.newMessageWithBody(record));
    }
```
If you think it's better, I'll replace it with "for of"

Reply:
Correct. The problem with `.forEach(async (record) =>` is that all emit calls will be made synchronously which can cause things to break when there are large numbers of records to emit.

Reply:
FIxed.